README.md

```diff
@@ -44,7 +44,10 @@ python3.11 -m pip install launch-cli==0.1.0
 ### Development Installation
 
 1. Clone this repository to your machine and enter the repository's directory.
-2. Create a new virtual environment and activate it with `python3.11 -m venv .venv && source .venv/bin/activate`.
+2. For Windows 11, for creating virtual environments with .
```

Comment:
Looks like there's something missing here

Reply:
Good catch, fixed it. Please review again.